tests/Rector/Class_/InitializeDefaultEntityCollectionRector/config/auto_import.php

```diff
@@ -2,11 +2,12 @@
 
 declare(strict_types=1);
 
+use Rector\Config\RectorConfig;
+
 use Rector\Core\Configuration\Option;
 use Rector\Doctrine\Rector\Class_\InitializeDefaultEntityCollectionRector;
-use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
-return static function (ContainerConfigurator $containerConfigurator): void {
+return static function (RectorConfig $containerConfigurator): void {
     $containerConfigurator->import(__DIR__ . '/../../../../../config/config.php');
     $parameters = $containerConfigurator->parameters();
     $parameters->set(Option::AUTO_IMPORT_NAMES, true);
```

Comment:
The parameters should be removed, and changed to direct methods of `RectorConfig`

Reply:
@TomasVotruba the `Naming` SetList can be used for it, ref https://github.com/rectorphp/rector-src/pull/2042

For note: after all packages configured with `RectorConfig`, we may need to create `Transform` Rector rule for it to change the method call.

Reply:
`RectorConfig` is a special class only rector use, and its change may be in the `tests` directory when developing custom rector rules, which containst multiple config, eg: https://github.com/laminas/laminas-servicemanager-migration/tree/0.4.x/test/Rector/Class_/ImplementsFactoryInterfaceToPsrFactoryRector/config. 

I think we can see if that's will be possible to create migrate rule for it.

Reply:
:+1: